packages/module/src/WarningModal/WarningModal.tsx

```diff
@@ -69,17 +92,36 @@ const WarningModal: React.FunctionComponent<WarningModalProps> = ({
       ouiaId={ouiaId}
       {...props}
     >
-      {children}
-      {withCheckbox ? (
-        <Checkbox
-          isChecked={checked}
-          onChange={(_event, value) => setChecked(value)}
-          label={checkboxLabel}
-          id="warning-modal-check"
-          className="pf-v6-u-mt-lg"
-          ouiaId={`${ouiaId}-confirm-checkbox`}
-        />
-      ) : null}
+      <Stack hasGutter>
+        <StackItem>{children}</StackItem>
+        <StackItem>
+          {confirmationInputProps ? (
+            <Flex direction={{ default: 'column' }} spaceItems={{ default: 'spaceItemsSm' }}>
+              <FlexItem>
+                {confirmationInputProps && confirmationInputLabel(deleteNameSanitized)}
```

Comment:
```suggestion
                {confirmationInputLabel(deleteNameSanitized)}
```
the check is probably not needed, since you have it above